project/terra_layer/views/layers.py

```diff
@@ -173,7 +173,9 @@ class SceneTreeAPIView(APIView):
                     ReportConfig.objects.prefetch_related(
                         Prefetch(
                             "report_fields",
-                            queryset=ReportField.objects.select_related("field"),
+                            queryset=ReportField.objects.select_related(
+                                "field"
+                            ).order_by("order"),
```

Comment:
If you define an order at model level, is it necessary to explicit order here ?

Reply:
Right, I removed it